docs/quick-start.md

```diff
@@ -3,22 +3,378 @@ title: Getting Started
 toc: true
 weight: 210
 ---
-# Quick Start Guide
+# Getting Started
 
-This quick start will demonstrate using Crossplane to deploy a portable MySQL
-database on a single cloud provider, Google Cloud Platform. It serves as an initial introduction to
+This guide will demonstrate using Crossplane to deploy a portable MySQL database
+on the Google Cloud Platform (GCP). It serves as an initial introduction to
 Crossplane, but only displays a small portion of its functionality.
 
-The general steps for this example are as follows:
+In this guide we will:
 
-1. [Install Crossplane and the GCP stack](install-crossplane.md) into your Kubernetes cluster. 
-1. [Configure GCP credentials](cloud-providers/gcp/gcp-provider.md) in Crossplane.
-1. [Provision a MySQL database](quick-start-gcp.md) on GCP. 
+1. [Install Crossplane and the GCP stack](#install-crossplane).
+1. [Configure Crossplane to connect to GCP](#configure-crossplane-to-connect-to-gcp).
+1. [Provision a MySQL instance](#provision-a-mysql-instance).
+1. [Enable Dynamic Provisioning of MySQL databases](#enable-dynamic-provisioning).
```

Comment:
Until I understood what you were saying, it was confusing having "**Enable** dynamic provisioning" _after_ "Provision a MySQL instance"

suggest ToC:

1. Install Crossplane
1. Connect to GCP
1. Provision a CloudSQL Instance
1. Dynamic Provisioning of MySQL Claims

Reply:
I see your point. I didn't go with your suggestion, but I played around with the wording of the headings and this "table of contents" (which is supposed to be more like a list of tasks that happens to link to titles, than a true ToC) to try to come up with something that is:

1. Succinct
1. Reasonably faithful to our terminology and concepts
1. Not so heavy on out terminology that it's meaningless to folks who don't already know it